core/main/src/broker/websocket_broker.rs

```diff
@@ -49,8 +49,13 @@ impl WebsocketBroker {
         let broker = BrokerSender { sender: tx };
         tokio::spawn(async move {
             if endpoint.jsonrpc {
-                let (mut ws_tx, mut ws_rx) =
-                    BrokerUtils::get_ws_broker(&endpoint.get_url(), None).await;
+                let resp = WebSocketUtils::get_ws_stream(&endpoint.get_url(), None).await;
+                if resp.is_err() {
+                    error!("Error connecting to websocket broker");
+                    tr.close();
+                    return false;
+                }
+                let (mut ws_tx, mut ws_rx) = resp.unwrap();
```

Comment:
Same feedback as earlier.
Avoid unsafe `unwrap()` and handle the error cases under `Err(__)` match arm